.github/workflows/contract-testing.yml

```
@@ -0,0 +1,271 @@
+          with open('openapi.json', 'r') as f:
+              spec = json.load(f)
+              print(f'✅ OpenAPI spec loaded: {spec[\"info\"][\"title\"]} v{spec[\"info\"][\"version\"]}')
+              print(f'   Base URL: $BASE_URL')
+              print(f'   Paths: {len(spec.get(\"paths\", {}))}')
+              print(f'   Schemas: {len(spec.get(\"components\", {}).get(\"schemas\", {}))}')
+          "
+          
+      - name: 🧪 Run contract tests - ${{ matrix.test-suite.name }}
+        run: |
+          echo "🧪 Running ${{ matrix.test-suite.name }}..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)